test/mocha/10-api.js

```diff
@@ -726,6 +877,145 @@ describe('bedrock-identity', function() {
           }]
         }, done);
       });
+      it('should not update identity in non-owned group via change set', done => {
+        var actor = actors.alpha;
+        var userName = '92c2f8ab-a5e1-4b62-88dd-c643518fb583';
+        var groupName = '7f61d0a9-052c-45ea-80a1-de656adef4ba';
+        var newIdentity = helpers.createIdentity(userName);
+        newIdentity.owner = actor.id;
+        newIdentity.sysResourceRole.push({
+          sysRole: 'bedrock-identity.regular',
+          generateResource: 'id'
+        });
+        var newGroup = helpers.createIdentity(groupName);
+        newGroup.type = ['Identity', 'Group'];
+        newGroup.owner = newIdentity.id;
+        newGroup.sysResourceRole.push({
+          sysRole: 'bedrock-identity.regular',
+          generateResource: 'id'
+        });
+        async.auto({
+          insertIdentity: callback => {
+            brIdentity.insert(actor, newIdentity, callback);
+          },
+          insertGroup: ['insertIdentity', callback => {
+            brIdentity.insert(newIdentity, newGroup, callback);
+          }],
+          update: ['insertGroup', callback => {
+            const updatedIdentity = newIdentity;
+            const changes = {
+              op: 'add',
+              value: {
+                memberOf: newGroup.id
+              }
+            };
+            brIdentity.update(
+              actor, updatedIdentity.id, {changes: changes}, err => {
+              should.exist(err);
+              err.name.should.equal('PermissionDenied');
+              err.details.sysPermission.should.equal(
+                'IDENTITY_UPDATE_MEMBERSHIP');
+              err.details.should.be.an('object');
+              callback();
+            });
+          }]
+        }, done);
+      });
+      it('should allow identity update w/ sysResourceRole for owned group',
```

Comment:
No, there's no way to do it with the legacy API, and we should really remove the membership tests too, but I didn't want to break anything that had been out there for a bit. I assume the `sysResourceRole` stuff was new enough to just rip out and not support any legacy things.